.php_cs.dist

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$finder = new PhpCsFixer\Finder();
+$config = new PhpCsFixer\Config('json-schema', 'json-schema style guide');
+$finder->in(__DIR__);
+
+/* Based on ^2.1 of php-cs-fixer */
+$config
+    ->setRules([
+        // default
+        '@PSR2' => true,
```

Comment:
Is there a specific need or requirement that phpcodesniffer solves that php-cs-fixer does not? If travis integration is the only thing, then I would like to point out that php-cs-fixer could just as easily be integrated into the Travis process. I just did not do it right away as not everyone is always in favor of this. E.g. at @composer we usually let style issues slip through in PR's because we do not want those to be a barrier for contributors. We run php-cs-fixer occasionally over the codebase ourselves manually.